build/release/Makefile

```
@@ -184,6 +186,10 @@ build.all.images: build.image.$(1).$(2)
 publish.image.$(1).$(2):
 	@$(DOCKERCMD) push $(1)/ceph-$(2):$(VERSION)
 	@$(DOCKERCMD) push $(1)/ceph-$(2):$(BRANCH_NAME)
+	@echo === cosign sign $(1)/ceph-$(2):$(VERSION)
+	@cosign sign --yes $(1)/ceph-$(2):$(VERSION)
```

How about removing the `@` prefix from all the cosign commands? Then we should see the command being executed, and you can also remove the new `echo` statements. 
```suggestion
	cosign sign --yes $(1)/ceph-$(2):$(VERSION)
```